modules/localgov_geo_address/templates/localgov-geo--address--embed.html.twig

```diff
@@ -14,8 +14,13 @@
  * @see template_preprocess_localgov_geo()
  */
 #}
+
 <article{{ attributes }}>
   {% if content %}
-    {{- content|without('label') -}}
+    {% if content.location['#object'].location|length == 0 %}
+      {{- content|without('label', 'location') -}}
```

Comment:
Missing indentation.

Reply:
I don't think the indentation is missing.